client/src/html.tsx

```diff
@@ -17,8 +19,16 @@ export default function HTML({
   postBodyComponents,
   preBodyComponents
 }: HTMLProps): JSX.Element {
+  const rtlDirectionAttribute = { dir: 'rtl' };
+  const isRtlLanguage: boolean = rtlLangs.includes(clientLocale);
+
   return (
-    <html id='__fcc-html' {...htmlAttributes} lang='en'>
+    <html
+      id='__fcc-html'
+      {...(isRtlLanguage && rtlDirectionAttribute)}
+      {...htmlAttributes}
+      lang='en'
```

Comment:
Should this be changed?

Reply:
Had similar thoughts, I discussed them in this comment https://github.com/freeCodeCamp/freeCodeCamp/issues/48101#issuecomment-1289431606. But there is no need to add logic to it somehow, `{...htmlAttribute}` is adding the other `lang`.

I was tunnel vision on making this logic `{...(isRtlLanguage && rtlDirectionAttribute)}`  part of `{...htmlAttributes}`. I didn't test it if we can delete `lang='en'`, and check the HTML element. Give me minutes, going to run the environment and check if there is `lang='en'`, or not

Reply:
Need not be this PR.

Reply:
I agree, it's there to please eslint, but it's useless, and deleting it is useless as well.

---

for documentation in case I forgot, we are using `htmlAttributes` in `Helmet` element, and passing `lang` in it. This way, we don't need to pass `lang` in the `html` element. But this make it difficult to change things, because it's almost impossible to find it without searching for `htmlAttributes`.

In the future. Ideally, we change the props to represent something like this. 

```ts
interface htmlProp {
lang: string,
dir?:string
} 

<html 
lang={language}
dir={direction}
></html>
...

<Helmet 
langauage={i18n.language} 
direction={i18n.language === RTLLanguage && RTLDirection}
/>
```

Here is the file

https://github.com/freeCodeCamp/freeCodeCamp/blob/3279ab161e8c0e4280b1deb4892ef5f76cdceb2e/client/src/components/app-mount-notifier.tsx#L30